core/src/main/java/hudson/slaves/CloudRetentionStrategy.java

```diff
@@ -47,6 +47,20 @@ public CloudRetentionStrategy(int idleMinutes) {
         this.idleMinutes = idleMinutes;
     }
 
+    /**
+     * @since FIXME
+     */
+    public int getIdleMinutes() {
+        return idleMinutes;
+    }
+
+    /**
+     * @since FIXME
+     */
+    public void setIdleMinutes(int idleMinutes) {
```

Comment:
It's perfectly fine because CloudRetentionStrategy is a part of the persistent object. If you create a custom setter, then you would have to bother about settings persistency once somebody changes the field via such API. And there is no documentation about "hey, you will have to save the Cloud instance on your own after the modification" in your method's Javadoc